template.yaml

```
@@ -35,6 +35,13 @@ Parameters:
     Default: "d9df08ac799f2859d42a588b415111314cf66d0ffd072195f33b921db966b440"
     ConstraintDescription: >-
       Must be the canonical ID for the AWS Synapse account
+  ObjectReadAccounts:
+    Type: String
+    Description: AWS account canonical IDs or email addresses to be granted object read permission
+    Default: "id=d9df08ac799f2859d42a588b415111314cf66d0ffd072195f33b921db966b440,emailaddress=dagutman@gmail.com"
```

does this mean that dagutman has object read access on every bucket?  is that intentional?

---

yes, the DSA needs read access to objects in the HTAN buckets

---

what if dagutman is no longer the `DSA`?  adding him as a default obscures this setting.  Would it not be better to not set a default but require the user who provisions the bucket to explicitly set one or more DSAs?

---

This is granting the DSA AWS account read access though, not dave gutman himself as a user? 
But sure, I can separate them. I also have the canonical ID now and can use it instead, if that's better than email address?

---

ahh, ok. the email confused me, thought it was giving perms to dagutman specifically .  if your giving perms to an entire account and it's unlikely to change then i guess no problem keeping it as-is.

---

yeah, it's for the account and unlikely to change
ok... thanks!